admin/src/portr_admin/services/user.py

```diff
@@ -14,6 +14,30 @@ class EmailFetchError(ServiceError):
     pass
 
 
+async def get_or_create_user(email: str):
+    is_first_user = not (await User.filter().exists())
+
+    if is_first_user:
+        # If this is the first user we can create them
+        await User.create(
+            email=email,
+            is_superuser=True,
+        )
+
+    user = await User.get_or_none(email=email)
+
+    if not user:
+        raise UserNotFoundError("User does not exist yet")
+
+    is_user_on_team = await TeamUser.filter(user__email=email).exists()
+
+    if not user.is_superuser and not is_user_on_team:
```

Comment:
This sort of fixes an existing issue - if a super user is created but then - say, because they get distracted watching Galaxy of Terror 😅 - gets logged out without creating a team, they're stuck.  They can't authenticate again that way because they aren't on a team, they never created one.

This makes it so a super user can always log back in and continue where they left off.

Reply:
Split this out to #28